packages/gafl-webapp-service/src/pages/order-complete/order-complete/order-complete.njk

```diff
@@ -21,40 +21,20 @@
           titleHtml: titleHtml,
           html: mssgs.order_complete_conf_1 + '<br><strong><span id="permission-number">' + data.permission.referenceNumber + '</span></strong><br /><br />' + mssgs.order_complete_conf_2 + '<br />' + data.startTimeStringTitle
         }) }}
-        {% if data.isPostalFulfilment %}
-            {% if data.contactMethod === data.howContacted['email'] %}
-                <p class="govuk-body">{{ mssgs.order_complete_email_conf }}</p>
-            {% endif %}
-            {% if data.contactMethod === data.howContacted['text'] %}
-                <p class="govuk-body">{{ mssgs.order_complete_text_conf }}</p>
-            {% endif %}
-            <p class="govuk-body">{{ mssgs.order_complete_post_conf }}</p>
-        {% else %}
-            {% if data.contactMethod === data.howContacted['email'] %}
-                <p class="govuk-body">{{ mssgs.order_complete_email_licence_1 }}</p>
-                <p class="govuk-body">{{ mssgs.order_complete_email_licence_2 }}</p>
-                <p class="govuk-body">{{ mssgs.order_complete_email_licence_3 }}</p>
-            {% endif %}
-            {% if data.contactMethod === data.howContacted['text'] %}
-                <p class="govuk-body">{{ mssgs.order_complete_text_licence_1 }}</p>
-                <p class="govuk-body">{{ mssgs.order_complete_email_licence_2 }}</p>
-                <p class="govuk-body">{{ mssgs.order_complete_email_licence_3 }}</p>
-            {% endif %}
-        {% endif %}
```

Comment:
Seconding what Nabeel's said - please check messages that can be deleted (you've done a few, but I think there are more redundancies?)